tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -123,3 +123,30 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: sharee searches shared project spaces files by content
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the GraphApi
+    And user "Alice" has shared a space "project-space" with settings:
+      | shareWith | Brian  |
+      | role      | viewer |
+    And user "Alice" has created a folder "spacesFolderWithFile/spacesSubFolder" in space "project-space"
+    And user "Alice" has uploaded a file inside space "project-space" with content "hello world from nepal" to "keywordAtStart.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "saying hello to the world" to "spacesFolderWithFile/keywordAtMiddle.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "nepal want to say hello" to "spacesFolderWithFile/spacesSubFolder/keywordAtLast.txt"
+    And user "Alice" has uploaded a file inside space "project-space" with content "namaste from nepal" to "hello.txt"
+    And using <dav-path-version> DAV path
+    When user "Brian" searches for "Content:hello" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | keywordAtStart.txt  |
+      | keywordAtMiddle.txt |
+      | keywordAtLast.txt   |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
can you make sure this test runs in stable-3.0 as well if not we can add the `@skipOnStable-3.0` tag directly here and backport it to stable. PLease run the test locally in stable branch and see if all the example passes

Reply:
okay